src/api-service/__app__/onefuzzlib/tasks/config.py

```diff
@@ -123,6 +123,19 @@ def check_target_exe(config: TaskConfig, definition: TaskDefinition) -> None:
         LOGGER.warning(err)
 
 
+def target_uses_input(config: TaskConfig) -> bool:
+    if config.task.target_options is not None:
+        for option in config.task.target_options:
+            if "{input}" in option:
+                return True
+    if config.task.target_env is not None:
+        for (key, value) in config.task.target_env.items():
+            if "{input}" in key or "{input}" in value:
```

Comment:
Do we really care if `{input}` is in the key? We don't expand it, right?

Reply:
addressed